### ISSUE TITLE
Add vs Put Method

### ISSUE THREAD

**Issue by jlee1007** [nus-cs2030/2122-s2#120]
Hi all, regarding L4 for the lab, I am confused about the difference between the "put" already constructed and "add" method that we are supposed to create. For add should we also be using put or should it be totally different? Thank you!

**Comment by mingshan2705** [comment]
Yup, the add method would make use of the put method.

**Comment by adityajirafe** [comment]
yep feel free to use the put methods defined/overriden in the diff classes such as module or student when defining the add method in your roster class

**Comment by undeser** [comment]
It depends on the way you want to implement it. For my design, ```put``` does not check if the Student already exists & will overwrite it regardless. Whereas for ```add```, it will check, if the Student does exist, it will literally add the Module in.
